### ISSUE TITLE
Fallback locale setting isn't being applied

### ISSUE THREAD

**Issue by brandonpittman** [pikax/vue-composable#167]
The I18n module should fallback to the fallback locale, but it isn't.

```js
locale: 'jp',
fallback: 'en',
messages: {
	en: { hello: 'Hello'},
	jp: {}
}
```

```html
{{ i18n.hello }} <!-- should be 'Hello' on jp locale -->
```

In the jp locale, it returns nothing. Am I missing something?

**Comment by pikax** [comment]
That's absolutely correct, there's a bug when the locale is different than the fallback it won't load the fallback.

An hack for that is:
```ts
 const { i18n, locale } = useI18n({
      locale: "en", // force it to load the fallback locale
      fallback: "en",
      messages: {
        en: { hello: "Hello" },
        jp: {}
      },
      notFoundFallback: true
    });
   // change to the correct locale
    locale.value = "jp";
```

that will make it work. I'm working on a fix and I should release in the next few hours.

**Comment by pikax** [comment]
This is fixed on [1.0.0-dev.15](https://github.com/pikax/vue-composable/releases/tag/v1.0.0-dev.15)

**Comment by brandonpittman** [comment]
Thank you!
